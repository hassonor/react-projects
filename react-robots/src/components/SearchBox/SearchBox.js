const SearchBox = ({ searchChange }) => {
  return (
    <div className="pa2">
      <input
        aria-label="Search Robots"
        className="pa3 ba b--green bg-lightest-blue"
        type="search"
        placeholder="Search Robots"
        onChange={searchChange}
      />
    </div>
  );
};

export default SearchBox;
