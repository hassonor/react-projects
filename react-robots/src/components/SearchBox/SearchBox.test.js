import React from "react";
import { shallow } from "enzyme";
import SearchBox from "./SearchBox";

it("expect to render SearchBox Component", () => {
  expect(shallow(<SearchBox />)).toMatchSnapshot();
});
