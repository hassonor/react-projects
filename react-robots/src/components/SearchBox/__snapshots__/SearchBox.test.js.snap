// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render SearchBox Component 1`] = `
<div
  className="pa2"
>
  <input
    aria-label="Search Robots"
    className="pa3 ba b--green bg-lightest-blue"
    placeholder="Search Robots"
    type="search"
  />
</div>
`;
